src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -673,9 +672,7 @@ public Block get(final PlayerInteractEvent e) {
 			@Override
 			@Nullable
 			public Direction get(final PlayerInteractEvent e) {
-				if (e.getBlockFace() != null)
-					return new Direction(new double[] {e.getBlockFace().getModX(), e.getBlockFace().getModY(), e.getBlockFace().getModZ()});
-				return Direction.ZERO; // Same as 'BlockFace.SELF' or literal 'at'
+				return new Direction(new double[] {e.getBlockFace().getModX(), e.getBlockFace().getModY(), e.getBlockFace().getModZ()});
```

Comment:
`getBlockFace` is marked as non-null, but has it always been?

Reply:
![image](https://user-images.githubusercontent.com/19924575/88005023-e6dce780-cb3a-11ea-9c0f-79f09da35976.png)
No mention of null here.